packages/generators/src/init-generator.ts

```diff
@@ -70,6 +73,24 @@ export default class InitGenerator extends CustomGenerator {
         }
 
         await handlers[this.template].questions(this, Question);
+
+        // Handle installation of prettier
+        try {
+            // eslint-disable-next-line node/no-extraneous-require
+            require.resolve('prettier');
+        } catch (err) {
+            const { installPrettier } = await Question.Confirm(
```

Comment:
We have some problems with init and no deps :disappointed: try to run `webpack init` without installed deps locally and look at result, I think we should solve it before merge it

Reply:
I known it is not directly problem in this PR, but if we want integration something with post install, we need solve it, otherwise it is very bad DX

Reply:
/cc @webpack/cli-team make sense to priority this problem and search fixes/workarounds/solutions

Reply:
Yup that something I feel to address as well. `webpack` and `webpack-cli` are now generally used as devDependencies, This adds noise of installing generators package as well as overriding package.json file during scaffolding. If we can make `webpack-cli` to be recommended for global installation (Like create-react-app, Vue CLI, etc), then we can enhance the DX here.